### PR TITLE
[2.2] new k8s versions 

### PR DESCRIPTION
**Pull request by kinarashah**
v1.13.12-rancher2-2, v1.14.10-rancher1-2, v1.15.12-rancher1-1

### CODE REVIEW

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 1]
```diff
@@ -1308,10 +1308,10 @@ var (
 			CoreDNS:                   m("coredns/coredns:1.2.6"),
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.0.0"),
 		},
-		// Enabled in Rancher v2.2.9
-		"v1.13.12-rancher1-1": {
+		// Enabled in Rancher v2.2.13
+		"v1.13.12-rancher2-2": {
 			Etcd:                      m("quay.io/coreos/etcd:v3.2.24-rancher1"),
-			Kubernetes:                m("rancher/hyperkube:v1.13.12-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.13.12-rancher2"),
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_defaults.go:1314]: Why is this `v1.13.12-rancher2`?

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 2]
```diff
@@ -1599,14 +1599,14 @@ var (
...
-		"v1.14.10-rancher1-0": {
+		// Enabled in Rancher v2.2.13
+		"v1.14.10-rancher1-2": {
 			Etcd:                      m("quay.io/coreos/etcd:v3.3.10-rancher1"),
 			Kubernetes:                m("rancher/hyperkube:v1.14.10-rancher1"),
-			Alpine:                    m("rancher/rke-tools:v0.1.50"),
-			NginxProxy:                m("rancher/rke-tools:v0.1.50"),
-			CertDownloader:            m("rancher/rke-tools:v0.1.50"),
-			KubernetesServicesSidecar: m("rancher/rke-tools:v0.1.50"),
+			Alpine:                    m("rancher/rke-tools:v0.1.52"),
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_defaults.go:1606]: Why are RKE tools changing?

File: apis/management.cattle.io/v3/k8s_defaults.go [hunk 3]
```diff
@@ -1809,14 +1809,14 @@ var (
...
+		// Enabled in Rancher v2.2.13
+		"v1.15.12-rancher1-1": {
 			Etcd:                      m("quay.io/coreos/etcd:v3.3.10-rancher1"),
-			Kubernetes:                m("rancher/hyperkube:v1.15.11-rancher1"),
-			Alpine:                    m("rancher/rke-tools:v0.1.50"),
-			NginxProxy:                m("rancher/rke-tools:v0.1.50"),
-			CertDownloader:            m("rancher/rke-tools:v0.1.50"),
-			KubernetesServicesSidecar: m("rancher/rke-tools:v0.1.50"),
+			Kubernetes:                m("rancher/hyperkube:v1.15.12-rancher1"),
+			Alpine:                    m("rancher/rke-tools:v0.1.56"),
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_defaults.go:1816]: Why are RKE tools changing?

File: apis/management.cattle.io/v3/k8s_defaults.go (hunk 1 above)
Comment (kinarashah) [reply apis/management.cattle.io/v3/k8s_defaults.go:1314]: it's based on the latest 1.13.x that exists currently https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/k8s_rke_system_images.go#L1097

File: apis/management.cattle.io/v3/k8s_defaults.go (hunk 2 above)
Comment (kinarashah) [reply apis/management.cattle.io/v3/k8s_defaults.go:1606]: https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/k8s_rke_system_images.go#L1454

File: apis/management.cattle.io/v3/k8s_defaults.go (hunk 3 above)
Comment (kinarashah) [reply apis/management.cattle.io/v3/k8s_defaults.go:1816]: This is new k8s version, so has the latest RKE tools introduced with it, https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.5/rke/k8s_rke_system_images.go#L1986

**Pull request merged**